.github/workflows/dependabot-auto-merge-go.yml

```diff
@@ -0,0 +1,39 @@
+# This action automatically merges dependabot PRs that update go dependencies (only patch and minor updates).
```

Comment:
Would make sense to rename the file to `dependabot-auto-merge-java.yml`.

Reply:
Good catch!